docs/en/getting-started/install.md

```diff
@@ -59,7 +59,7 @@ clickhouse-client # or "clickhouse-client --password" if you set up a password.
 
 </details>
 
-You can replace `stable` with `lts` or `testing` to use different [release trains](../faq/operations/production.md) based on your needs.
+You can replace `stable` with `lts` to use different [release trains](../faq/operations/production.md) based on your needs.
```

Comment:
Let's get rid of these strange "trains". In https://clickhouse.com/docs/en/faq/operations/production#how-to-choose-between-clickhouse-releases it "release kinds"

Reply:
Do you mean replacing `[release trains] ` with the `[release kinds]`?

Reply:
I also create this [PR](https://github.com/ClickHouse/clickhouse-docs/pull/361) to get rid of the "strange trains".